### ISSUE TITLE
Compatibility with Peddler

### ISSUE THREAD

**Issue by Pierre-Sassoulas** [Pierre-Sassoulas/grey-handling#25]
> Can you add compatibility with Peddler? Should be the same as Scrap.

**Comment by Pierre-Sassoulas** [comment]
The function `local function itemIsToBeSold(itemID, uniqueItemID)` is declared as local and is innaccessible.

**Comment by Pierre-Sassoulas** [comment]
The Peddler's dev accepted to make it public 

https://www.curseforge.com/wow/addons/peddler#c139

>How do I go about making said function public so that you can access it?

> I think you only have to remove the "local" keyword. But in order to do something cleaner you could do : `local function itemIsToBeSold(itemID, uniqueItemID)` => `function Peddler.itemIsToBeSold(itemID, uniqueItemID)`. So we have a Peddler namespace attached to it and it's easier to understand what's happening.

**Comment by Pierre-Sassoulas** [comment]
Will be possible once https://github.com/Cartas/Peddler/pull/4 is merged.

**Comment by Jinnai** [comment]
It's merged. Any updates?

**Comment by Pierre-Sassoulas** [comment]
I think I made a mistake when I said that you only needed to add a namespace to make it public. I can't access the Peddler function right now. So I installed a dev version of Peddler locally and I'm going to make a merge request to peddler when I manage to make it works together. I had to refactor my options before being able to choose between Scrap, Peddler and my addon for determining what is junk, and that's what I just did in 1.5.0 so I will be able to test Peddler integration soon.
